packages/replay/src/util/createPerformanceEntries.ts

```diff
@@ -13,11 +13,21 @@ import type {
   ResourceData,
 } from '../types';
 
+// from rrweb1
+interface Mirror {
```

Comment:
l: should we move this to types/rrweb ?

Reply:
Also, what about shaping this interface with only the subset of `Mirror` we actually need? So if a mirror implementation changes in rrweb, it should only fail if the stuff we actually depend on is not there anymore?